ethcore/src/miner/miner.rs

```diff
@@ -136,6 +136,8 @@ pub struct MinerOptions {
 	/// will be invalid if mined.
 	pub infinite_pending_block: bool,
 
+	/// Prioritized Local Addresses
+	pub tx_queue_locals: Vec<Address>,
```

Comment:
Should we consider some kind of fast `HashSet` for this? Not sure how many accounts people plan to have here, but using a `Vec` might impact performance drastically for larger amounts of accounts.